.github/GUIDE.md

```diff
@@ -0,0 +1,51 @@
+# Step-by-Step Guide for Contributing to a Splunk SOAR App
+
+## Step 1: Create a GitHub Issue
```

Comment:
we shouldn't request contributors make an issue if they're also making a PR as it just ends up making it harder to filter out issues w/o PRs and doesn't provide any extra information (you can just write it in the PR). we removed this requirement (issue before PR) from the monorepo earlier this year

Reply:
makes sense, I've removed this step!